### PR TITLE
Functional CopyToClipboard and CreatePassPhrase

### PR DESCRIPTION
**Pull request by bgptr**
This diff refactors `CopyToClipboard` and `CreatePassPhrase` to functional components using hooks and CSS modules.
 Also, it includes automated tests.

Part of #2438 and #2439

### CODE REVIEW

File: app/components/shared/CopyToClipboard/CopyToClipboard.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import copy from "clipboard-copy";
+import { FormattedMessage as T } from "react-intl";
+import styles from "./CopyToClipboard.module.css";
+import { classNames } from "pi-ui";
+import { useState } from "react";
+
+const CopyToClipboard = ({ textToCopy, className }) => {
+  const [isSuccessHidden, setIsSuccessHidden] = useState(true);
+
+  function onClick() {
```
Comment (amassarwi) [inline app/components/shared/CopyToClipboard/CopyToClipboard.js:11]: arrow function

File: app/components/shared/CopyToClipboard/CopyToClipboard.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if (copy(textToCopy)) setIsSuccessHidden(false);
+  }
+
+  function onMouseLeave() {
```
Comment (amassarwi) [inline app/components/shared/CopyToClipboard/CopyToClipboard.js:15]: arrow function

File: app/components/shared/CopyToClipboard/CopyToClipboard.module.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.clipboardBox {
+  position: relative;
+  display: inline-block;
+}
+.copyToClipboardIcon {
```
Comment (amassarwi) [inline app/components/shared/CopyToClipboard/CopyToClipboard.module.css:5]: could deleted the `copyToClipboard` prefix from this and other classes too.

File: app/components/shared/CreatePassPhrase/CreatePassPhrase.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+const CreatePassPhrase = ({ onChange, onSubmit, ...props }) => {
+  const [passPhrase, setPassPhrase] = useState("");
+  const [passPhraseVerification, setPassPhraseVerification] = useState("");
+  const [hasFailedAttempt, setHasFailedAttempt] = useState(false);
+
+  const isValid = useCallback(() => {
+    return !!passPhrase && passPhrase === passPhraseVerification;
+  }, [passPhrase, passPhraseVerification]);
+
+  function onKeyDown(e) {
```
Comment (amassarwi) [inline app/components/shared/CreatePassPhrase/CreatePassPhrase.js:13]: arrow func

File: app/components/shared/CreatePassPhrase/CreatePassPhrase.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    // Enter key
+    if (e.keyCode == 13) {
+      e.preventDefault();
+      onSubmit?.();
+    }
+  }
+
+  function setPassPhraseAndHasFailedAttempt(passPhrase) {
```
Comment (amassarwi) [inline app/components/shared/CreatePassPhrase/CreatePassPhrase.js:21]: arrow func

File: app/components/shared/CreatePassPhrase/CreatePassPhrase.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    setHasFailedAttempt(true);
+    setPassPhrase(passPhrase);
+  }
+
+  useEffect(() => {
```
Comment (amassarwi) [inline app/components/shared/CreatePassPhrase/CreatePassPhrase.js:26]: i would move this to be above functions declarations if possible.

**Review by amassarwi — COMMENTED** [review]
Code looks good, left some minors.

Also, please reference #2439 when you refactor `.less` files to css modules.

**Comment by bgptr** [comment]
@amassarwi Thank you for your review comments. I've fixed them.

**Review by amassarwi — COMMENTED** [review]
Please change all components/jsx files extention from .js  to .jsx

**Comment by bgptr** [comment]
@amassarwi done

**Review by alexlyp — APPROVED** [review]

**Pull request merged**